pydantic/_internal/_generate_schema.py

```diff
@@ -2304,6 +2312,8 @@ def _apply_model_serializers(
 }
 
 
+# TODO V3: this function is only used for deprecated decorators. It should
+# be removed once we drop support for those.
```

Comment:
Looks like this is also used in some other places in the file - just wanted to make sure you were aware of that...

Reply:
Yes, it is used for deprecated decorators